2021-04-06 10:31:49,193 INFO    MainThread:21919 [wandb_init.py:_log_setup():292] Logging user logs to /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_103149-2ej217de/logs/debug.log
2021-04-06 10:31:49,193 INFO    MainThread:21919 [wandb_init.py:_log_setup():293] Logging internal logs to /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_103149-2ej217de/logs/debug-internal.log
2021-04-06 10:31:49,194 INFO    MainThread:21919 [wandb_setup.py:_flush():68] setting env: {}
2021-04-06 10:31:49,194 INFO    MainThread:21919 [wandb_setup.py:_flush():68] setting user settings: {}
2021-04-06 10:31:49,194 INFO    MainThread:21919 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2021-04-06 10:31:51,247 INFO    MainThread:21919 [wandb_run.py:_console_start():1265] atexit reg
2021-04-06 10:31:51,248 INFO    MainThread:21919 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2021-04-06 10:31:51,250 INFO    MainThread:21919 [wandb_run.py:_redirect():1138] Redirecting console.
2021-04-06 10:31:51,250 INFO    MainThread:21919 [redirect.py:install():196] install start
2021-04-06 10:31:51,250 INFO    MainThread:21919 [redirect.py:install():211] install stop
2021-04-06 10:31:51,251 INFO    MainThread:21919 [redirect.py:install():196] install start
2021-04-06 10:31:51,251 INFO    MainThread:21919 [redirect.py:install():211] install stop
2021-04-06 10:31:51,251 INFO    MainThread:21919 [wandb_run.py:_redirect():1182] Redirects installed.
2021-04-06 10:36:32,981 INFO    MainThread:21919 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 1
2021-04-06 10:36:33,069 INFO    MainThread:21919 [wandb_run.py:_restore():1210] restore
2021-04-06 10:36:33,069 INFO    MainThread:21919 [redirect.py:uninstall():215] uninstall start
2021-04-06 10:36:33,069 INFO    MainThread:21919 [redirect.py:_stop():264] _stop: stdout
2021-04-06 10:36:33,070 INFO    MainThread:21919 [redirect.py:_stop():270] _stop closed: stdout
2021-04-06 10:36:33,070 INFO    stdout    :21919 [redirect.py:_pipe_relay():119] relay done saw last write: stdout
2021-04-06 10:36:33,070 INFO    stdout    :21919 [redirect.py:_pipe_relay():135] relay done done: stdout
2021-04-06 10:36:33,071 INFO    MainThread:21919 [redirect.py:_stop():276] _stop joined: stdout
2021-04-06 10:36:33,071 INFO    MainThread:21919 [redirect.py:_stop():278] _stop rd closed: stdout
2021-04-06 10:36:33,071 INFO    MainThread:21919 [redirect.py:uninstall():219] uninstall done
2021-04-06 10:36:33,071 INFO    MainThread:21919 [redirect.py:uninstall():215] uninstall start
2021-04-06 10:36:33,071 INFO    MainThread:21919 [redirect.py:_stop():264] _stop: stderr
2021-04-06 10:36:33,071 INFO    MainThread:21919 [redirect.py:_stop():270] _stop closed: stderr
2021-04-06 10:36:33,071 INFO    stderr    :21919 [redirect.py:_pipe_relay():119] relay done saw last write: stderr
2021-04-06 10:36:33,071 INFO    stderr    :21919 [redirect.py:_pipe_relay():135] relay done done: stderr
2021-04-06 10:36:33,072 INFO    MainThread:21919 [redirect.py:_stop():276] _stop joined: stderr
2021-04-06 10:36:33,072 INFO    MainThread:21919 [redirect.py:_stop():278] _stop rd closed: stderr
2021-04-06 10:36:33,072 INFO    MainThread:21919 [redirect.py:uninstall():219] uninstall done
2021-04-06 10:36:36,635 ERROR   MainThread:21919 [wandb_run.py:_atexit_cleanup():1256] Problem finishing run
Traceback (most recent call last):
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 1246, in _atexit_cleanup
    self._on_finish()
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 1367, in _on_finish
    self.history._flush()
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/sdk/wandb_history.py", line 49, in _flush
    self._callback(row=self._data, step=self._step)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 630, in _history_callback
    self._backend.interface.publish_history(row, step)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/interface/interface.py", line 171, in publish_history
    item.value_json = json_dumps_safer_history(v)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/util.py", line 529, in json_dumps_safer_history
    return json.dumps(obj, cls=WandBHistoryJSONEncoder, **kwargs)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/util.py", line 496, in default
    obj, converted = json_friendly(obj)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/util.py", line 365, in json_friendly
    return obj.item(), True
RuntimeError: CUDA error: device-side assert triggered
